README.md

```diff
@@ -143,6 +153,10 @@ By default, trace telemetry submits:
 - **rendered message** in trace's standard *message* property.
 - **severity** in trace's standard *severityLevel* property.
 - **timestamp** in trace's standard *timestamp* property.
+- **operation id** from `operationId` property, or the `LogEvent.TraceId` property.
```

Comment:
Looking at the inconsistent casing here, perhaps the code in the background should accept both `operationId` and `OperationId`, as well as `version` and `Version`, but the README should only document the (added) `OperationId` and `Version` properties? 

Not something needed for this PR but might be worth doing to keep the API feeling consistent.

Reply:
No problem, done. I used this for all externally provided properties, with an option to enable case-sensitivity